cisco_aci/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+## 11.1.0 / 2023-08-18 / Agent 7.48.0
```

Comment:
🤔 this seems weird

Reply:
Looks like I got something wrong during conflict resolution. On it